src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    operate: operate,
+    add: add,
+    subtract: subtract,
+    multiply: multiply,
+    divide: divide,
+    reset: reset,
+    result: 0,
+  };
+
+  function operate(operation, number = 0) {
+    operation(number);
```

Comment:
The 'this' keyword is not bound to any object inside the 'operate' function. Consider using bind, call or apply method to bind 'this' to the calculator object.